src/docs/asciidoc/chapters/index.adoc

```diff
@@ -0,0 +1,3 @@
+:page-title: My Book
```

Comment:
☝🏻 Define a title on the page.
This title will be used as the document title on the generated HTML page: `My Book :: InfoQ Mini Book`

Reply:
Excellent! I like this, but for the Angular Mini-Book, I think it'd render `The Angular Mini Book :: The Angular Mini Book`. I could see using `Home` as the title, but ideally, there'd be a way to just render the title of the book on the main landing page.

Reply:
I think the following should work;

```
= The Angular Mini Book
:page-title: Home
```

The document title on the generated HTML page will be `Home :: The Angular Mini Book` while the title (`<h1>`) on the HTML page will be `The Angular Mini Book`.